catalog/app/components/SearchResults/SearchResults.js

```diff
@@ -613,6 +613,9 @@ export const handleErr = (retryUrl) => (e) => (
           supported query syntax
         </StyledLink>
         .
+        {!!retryUrl && (
+          <> You can also click RETRY to try a simplified version of your query.</>
```

Comment:
Just curious, can we replace `<></>` with quotes? Or it doesn't work such way?

Reply:
yep, in this case we can, bc there's no nested react elements